test_conformance/pipes/test_pipe_info.cpp

```diff
@@ -52,29 +52,37 @@ int test_pipe_info( cl_device_id deviceID, cl_context context, cl_command_queue
 
     if(pipe_depth != returnVal)
     {
-        log_error( "Error in clGetPipeInfo() check of pipe max packets\n" );
-        return -1;
+        test_fail("Error in clGetPipeInfo() check of pipe max packets\n");
     }
     else
     {
         log_info( " CL_PIPE_MAX_PACKETS passed.\n" );
     }
 
-    err = create_single_kernel_helper_with_build_options(context, &program, &kernel, 1, (const char**)&pipe_kernel_code, "pipe_kernel", "-cl-std=CL2.0 -cl-kernel-arg-info");
-    test_error_ret(err, " Error creating program", -1);
+    err = create_single_kernel_helper_with_build_options(
+        context, &program, &kernel, 1, &pipe_kernel_code, "pipe_kernel",
+        "-cl-std=CL2.0 -cl-kernel-arg-info");
+    test_error_ret(err, "Error creating program", TEST_FAIL);
```

Comment:
Minor/optional: This pattern can  be shortened to `test_err_fail(err, "<error message>")`.

(There's another case a few lines below also.)

Reply:
I introduced the `test_error_fail` macro so should have really remembered to use it here! (Except of course this change is quite a bit older than that macro and had sat unreviewed for a long time, and seemed fine as it still rebased cleanly... so my apologies for that.)